scripts/eurostat/health_determinants/social_environment/preprocess.py

```diff
@@ -0,0 +1,501 @@
+                    sv_name = sv_name + prop + ", "
+                elif "Relatives" in prop:
+                    benificiary = "\nsocialSupportBeneficiaryType: dcs:" + prop
+                    sv_name = sv_name + prop + ", "
+                elif "AtLeastOnceAWeek" in prop:
+                    frequency = "\nactivityFrequency: dcs:" + prop
+                    sv_name = sv_name + prop + ", "
+            # Making the changes to the SV Name,
+            # Removing any extra commas, with keyword and
+            # adding Population in the end
+            sv_name = sv_name.replace(", Among", " Among")
+            sv_name = sv_name.rstrip(', ')
+            sv_name = sv_name.rstrip('with')
+            # Adding spaces before every capital letter,
+            # to make SV look more like a name.
+            sv_name = re.sub(r"(\w)([A-Z])", r"\1 \2", sv_name)
+            sv_name = "name: \"" + sv_name + " Population\""
+            sv_name = sv_name.replace('AWeek',
+                                      'A Week').replace('ACitizen', 'A Citizen')
+
+            final_mcf_template += _MCF_TEMPLATE.format(pv1=sv,
+                                                       pv2=sv_name,
+                                                       pv3=denominator,
+                                                       pv4=education,
+                                                       pv5=residence,
+                                                       pv6=assist,
+                                                       pv7=lev_perc,
+                                                       pv8=gender,
+                                                       pv9=countryofbirth,
+                                                       pv10=citizenship,
+                                                       pv11=lev_limit,
+                                                       pv12=benificiary,
+                                                       pv13=frequency) + "\n"
+
+        # Writing Genereated MCF to local path.
+        with open(self._mcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+        # pylint: enable=R0914
+        # pylint: enable=R0912
+        # pylint: enable=R0915
+
+    def _generate_tmcf(self) -> None:
+        """
+        This method generates TMCF file w.r.t
+        dataframe headers and defined TMCF template.
+        Args:
+            None
+        Returns:
+            None
+        """
+        # Writing Genereated TMCF to local path.
+        with open(self._tmcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(_TMCF_TEMPLATE.rstrip('\n'))
+
+    def process(self):
+        """
+      This Method calls the required methods to generate
+      cleaned CSV, MCF, and TMCF file
+      """
+        final_df = pd.DataFrame(columns=['time','geo','SV','observation',\
+            'Measurement_Method'])
```

Comment:
Can we have consistent naming convention. E.g., all column names in small case and having underscore incase if it is more than one word column name.
Try removing backslash '\' and let lint take care of formatting.

Reply:
Done the changes.